crawl-ref/source/areas.cc

```diff
@@ -612,6 +612,7 @@ static int _mons_class_halo_radius(monster_type type)
     case MONS_SERAPH:
         return 7; // highest rank among sentient ones
     case MONS_HOLY_SWINE:
+    case MONS_SUN_MOTH:
```

Comment:
Sort of funny to have a halo monster in the ghost moth branch. Very antisynergy. (Also a little antisynergistic with EV monsters generally.)

As long as it's still a dangerous monster, though, that's probably fine..? Just don't want folks luring this around.